### PR TITLE
[master] ZIL-5488: acceptance test fixes

### PR DESCRIPTION
**Pull request by rrw-zilliqa**
## Description

A few modifications to allow robotic invocation of the Evm acceptance tests.

## Backward Compatibility

- [X] This is not a breaking change
- [ ] This is a breaking change

## Review Suggestion

Look at the diff and run the tests.

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: tests/EvmAcceptanceTests/test/otterscan/ots_getTransactionBySenderAndNonce.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ describe(`Otterscan api tests: ${METHOD}`, function () {
     const accounts = Array.from({length: ACCOUNTS_COUNT}, (v, k) =>
       ethers.Wallet.createRandom().connect(ethers.provider)
     );
-
+
```
Comment (its-saeed) [inline tests/EvmAcceptanceTests/test/otterscan/ots_getTransactionBySenderAndNonce.ts:27]: Please run `npm run lint`

**Review by its-saeed — APPROVED** [review]

File: tests/EvmAcceptanceTests/test/otterscan/ots_getTransactionBySenderAndNonce.ts (hunk 1 above)
Comment (its-saeed) [reply tests/EvmAcceptanceTests/test/otterscan/ots_getTransactionBySenderAndNonce.ts]: I'm sure you believe that keeping the project clean is part of delivering it in the long term. `npm run lint` as you know doesn't prevent you from going forward. It fixes the code style problems automatically in a few seconds.

Comment (rrw-zilliqa) [reply tests/EvmAcceptanceTests/test/otterscan/ots_getTransactionBySenderAndNonce.ts]: And yet, it's a week later and we're still not merged :-(

**Review by its-saeed — DISMISSED** [review]

**Review by mauromedda — APPROVED** [review]

**Comment by its-saeed** [comment]
`Error HH8: There's one or more errors in your config file.` Your config file seems to be invalid.

**Review by mauromedda — APPROVED** [review]

**Pull request merged**